src/applications/personalization/common/components/devtools/DevTools.jsx

```diff
@@ -14,35 +14,10 @@ export const DevTools = ({
   alwaysShowChildren = true,
   showIcon = true,
   panel = false,
-  children,
+  children = null,
 }) => {
   const [visible, setVisible] = useState(false);
 
-  if (!window?.showDevTools) {
-    window.showDevTools = () => {
-      document.dispatchEvent(
-        new CustomEvent('devToolsActivate', { detail: true }),
-      );
-    };
-  }
-
-  if (!window?.hideDevTools) {
-    window.hideDevTools = () => {
-      document.dispatchEvent(
-        new CustomEvent('devToolsActivate', { detail: false }),
-      );
-    };
-  }
-
-  document.addEventListener('devToolsActivate', event => {
-    if (event.detail === true) {
-      setVisible(true);
-      return;
-    }
-
-    setVisible(false);
-  });
-
```

Comment:
I removed this from the devtools because it doesn't really serve a purpose and wasnt being used from my original POC ideas.